packages/ui-components/package.json

```diff
@@ -28,7 +28,6 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.47",
     "@material-ui/pickers": "^3.2.10",
-    "@material-ui/styles": "v4.2.0",
```

Comment:
This looks like it was added to make the data grid work - might be worth checking with @chris-bes about that?

Reply:
I was indeed! But we chatted and he said it was all good to remove :+1: (and I checked and the data-grid is still looking good)